apps/backend/src/plugins/authorization/authorization.plugin.ts

```diff
@@ -69,16 +69,13 @@ const extractUserFromRequest = async (
 
 const checkIsWhiteRoute = (url: string, whiteRoutes: string[]): boolean => {
 	const regex = /^\/api\/v\d+(\/.+)$/;
-	const match = regex.exec(url);
-	const [, route] = match ?? [];
+	const isAPIRoute = regex.test(url);
 
-	if (!route) {
+	if (!isAPIRoute) {
 		return true;
 	}
 
-	const ROUTE_WITH_QUERY_INDEX = 0;
-
-	const routeWithoutQuery = route.split("?")[ROUTE_WITH_QUERY_INDEX] as string;
+	const [routeWithoutQuery] = url.split("?") as [string];
 
 	return whiteRoutes.includes(routeWithoutQuery);
```

Comment:
```suggestion
	const [routeWithoutQuery] = route.split("?");

	return whiteRoutes.includes(routeWithoutQuery as string);
```
Let's do it in this way :)